scripts/mkcloud

```diff
@@ -539,12 +539,32 @@ function setupadmin()
     fi
 }
 
+function wait_for_node_shutdown()
+{
+    wait_for 150 1 "virsh domstate $1 | grep shut.off" '$1 node to shut down'
+}
+
+function remove_snapshot_volume()
+{
+    lvremove -f $1
```

Comment:
This could remove any LV. How about checking if the volume really is a snapshot volume first?